src/app.js

```diff
@@ -1 +1,44 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from "axios"
+//Stap 1. Installeer en importeer Axios;
+console.log('Hallo daar!');
+
+// Stap 3. Schrijf een asynchrone functie die, met behulp van Axios, een GET-request maakt naar het juiste endpoint. Log de response in de console en bestudeer de data goed.
+async function getCountries(){
+    const countryList = document.getElementById("country-list");
+    try {
+        const result = await axios.get('https://restcountries.com/v2/all');
+        const countries = result.data;
+        // console.log(result.data);
+
+        countries.sort((a, b)=>{
+            return a.population - b.population;
+        });
+        result.data.map((country) => {
+            const countryItem = document.createElement("li");
+            countryItem.innerHTML = `
+            <img src="${country.flag}" alt="vlaggetje van landje" class="flag"/>
```

Comment:
Strikt genomen had ik de naam van het land opgenomen in de alt-waarde. Nu zou een slechtziende met zijn/haar screenreader niet weten welk vlaggetje er staat